cppcache/integration/framework/Cluster.cpp

```diff
@@ -42,27 +42,39 @@ void Locator::start() {
       .withPort(locatorAddress_.port)
       .withMaxHeap("256m")
       .withJmxManagerPort(jmxManagerPort_)
-      .withHttpServicePort(0)
-      .execute();
+      .withHttpServicePort(0);
+
+  if (!cluster_.getClasspath().empty()) {
```

Comment:
Why check this, just set the empty string on the locator class. All the lines above don't care about empty strings.